draft-pauly-ohai-svcb-config.md

```diff
@@ -59,12 +60,16 @@ and applies local network resolution policies via mechanisms like Discovery
 of Designated Resolvers ({{!DDR=I-D.draft-ietf-add-ddr}}. Clients
```

Comment:
We may want to elaborate on the deployment model assumed by this draft for DDR and DNR. For example, (1) the DNS server needs to have public access for the oblivious proxy to access it (2) client has an prior agreement with the oblivious proxy that the name and path of the oblivous target is encoded using a particular proxy URI template.

Reply:
Added text at the start of the oblivious DNS section for this!

Reply:
Thanks.